.github/workflows/ci.yml

```diff
@@ -14,16 +14,12 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
```

Comment:
@EjPlatzer are these changes intentional, or is this something that needs to be reverted?  I believe these were your changes, but just wanted to check.